app/src/main/res/layout/fragment_hangout_details.xml

```diff
@@ -0,0 +1,125 @@
+                android:layout_marginTop="40dp"
+                android:fontFamily="@font/roboto_bold"
+                android:textColor="@color/black"
+                android:textSize="24sp"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                tools:text="Dinner with friends" />
+
+            <TextView
+                android:id="@+id/tvHangoutDetailsTime"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:fontFamily="@font/roboto"
+                android:textColor="@color/black"
+                android:textSize="18sp"
+                app:layout_constraintStart_toStartOf="@+id/tvHangoutDetailsDate"
+                app:layout_constraintTop_toBottomOf="@+id/tvHangoutDetailsDate"
+                tools:text="16:15" />
+
+            <TextView
+                android:id="@+id/tvHangoutDetailsLocation"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="5dp"
+                android:fontFamily="@font/roboto"
+                android:textColor="@color/black"
+                android:textSize="18sp"
+                app:layout_constraintStart_toStartOf="@+id/tvHangoutDetailsAlias"
+                app:layout_constraintTop_toBottomOf="@+id/tvHangoutDetailsAlias"
+                tools:text="Menlo Park" />
+
+            <fragment
+                android:id="@+id/detailsMapFragment"
+                android:name="com.google.android.gms.maps.SupportMapFragment"
+                android:layout_width="0dp"
+                android:layout_height="300dp"
+                android:layout_marginTop="@dimen/content_margin"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/tvHangoutDetailsTime"
+                tools:context=".homeScreen.fragments.HangoutDetailsFragment" />
+
+            <ImageButton
+                android:id="@+id/btnCopyClipboard"
+                android:layout_width="60dp"
+                android:layout_height="60dp"
+                android:backgroundTint="@color/transparent"
+                android:scaleType="fitXY"
+                android:src="@drawable/ic_clipboard_icon"
+                app:layout_constraintEnd_toEndOf="@+id/tvHangoutDetailsHangoutCode"
+                app:layout_constraintStart_toStartOf="@+id/tvHangoutDetailsHangoutCode"
+                app:layout_constraintTop_toBottomOf="@+id/tvHangoutDetailsHangoutCode" />
+
+            <TextView
+                android:id="@+id/tvHangoutDetailsDate"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:fontFamily="@font/roboto"
+                android:textColor="@color/black"
+                android:textSize="18sp"
```

Comment:
same with these text sizes - extract to dimen.  
Themes can be useful also to have consistent text/views and can be applied easily.  Then you can change the look of all views from a single location.